integration/templates/next-app-router/next.config.js

```diff
@@ -6,6 +6,7 @@ const nextConfig = {
   experimental: {
     serverActions: true,
   },
+  outputFileTracingRoot: '/',
```

Comment:
By default, Turbopack doesn't resolve files outside of the project root. We use `pnpm link` to install dependencies, which exist outside of the root. We set the `outputFileTracingRoot` to the root `/` directory to allow Turbopack to resolve dependency files from anywhere on disk.